Solutions/Farsight DNSDB/Playbooks/DNSDB_Co_Located_Hosts/azuredeploy.json

```diff
@@ -0,0 +1,748 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "comments": "This playbook uses the Farsight DNSDB connector to automatically enrich incidents generated by Sentinel. Learn more about the integration via the https://docs.microsoft.com/en-us/connectors/farsightdnsdb/ or visit https://www.farsightsecurity.com/about-farsight-security/contacts/ to request a trial key.",

```

Comment:
Links should not contain locale. Because of that validation is failing. Please update the link as **https://docs.microsoft.com/connectors/farsightdnsdb/**